src/api_proxy/service_control/request_info.h

```diff
@@ -45,6 +45,28 @@ inline const char* ToString(Protocol p) {
 
 }  // namespace protocol
 
+namespace identity {
+
+// Indicates whether the API key is verified by SC Check.
+enum ApiConsumerIdentity {
```

Comment:
I still prefer ApiKeyStatus.   Could you elaborate on the name conflicting issue?

Reply:
Because the API Key can be valid, but the consumer service may not have activated the service. In that case, we should not fill in the credential id. It would be conflicting to say `api_key_status: invalid` when the API Key is actually valid.

Reply:
How about adding a new enum state: NOT_CHECKED,  INVALID,  VALID, NOT_ENABLED.  
Can we still call it ApiKeyStatus or State?

Reply:
Sounds good, I'll use `ApiKeyState`